megatron/core/optimizer/optimizer.py

```diff
@@ -3,6 +3,7 @@
 """Megatron optimizer."""
 
 import copy
+import logging
```

Comment:
Not going to block approval on this, but is this needed? (if not, remove in a different PR)

Reply:
It's been referenced on L347 and L367 for a while. No one tried to use those methods before today, so the lack of `logging` was not noticed.